lib/route.js

```diff
@@ -267,7 +267,7 @@ function buildRouting (options) {
           fourOhFour.setContext(this, context)
 
           if (opts.schema) {
-            context.schema = normalizeSchema(context.schema)
+            context.schema = normalizeSchema(context.schema, this[kOptions])
```

Comment:
Here we could read the `initialConfig` if I'm not wrong

Reply:
How would I access that from here? This was why I had assigned the the property on the options object in the root `fastify.js` module.
`initialConfig` wouldn't get updated with what the user passes in would it?

Reply:
it should be `this.initialConfig` since `this` is the server instance